### ISSUE TITLE
Fails to build

### ISSUE THREAD

**Issue by TheMattSchiller** [haugene/docker-transmission-openvpn#272]
ive tried on several machines with the completely vanilla setup and docker build is failing with the following error:

failed to process "\"TRANSMISSION_ALT_SPEED_DOWN": unexpected end of statement while looking for matching double-quote

**Comment by haugene** [comment]
Well there seems to be an extra double quote in front of that env variable, if it's exactly what you printed here. Can't see it's like that in the master or dev branch of the repository. Have you made any changes there?

**Comment by lordhippo93** [comment]
i think i had the same problem. for me, i was able to build it after i added 1 more " to every line.

from: "TRANSMISSION_ALT_SPEED_DOWN=50" \
to: ""TRANSMISSION_ALT_SPEED_DOWN=50"" \

after that, it built.

**Comment by TheMattSchiller** [comment]
Interesting. I was able to make it build after removing the ENV section, then I used a DockerEnv file with said settings

**Comment by haugene** [comment]
The [automated builds](https://hub.docker.com/r/haugene/transmission-openvpn/builds/) on Docker Hub has not failed for 5 months, and that was just briefly with an error I introduced. Which means that the pushes made to github should all be OK.

If an error occurs on your local machine, while the repository builds on the Docker Hub, I would say that it's likely something that has happened during your clone process or that you've modified the code.
This might have to do with line endings if you're working on a Windows machine. Does git checkout linux-style line endings or convert them during a clone? If it's converted then the build might fail because the files are added to the container and it expects linux style ending, etc...

Glad to see you found a way to make it build, just wanted to add some extra information for future reference if others encounter problems building. Closing this now.
